app/src/main/java/com/peacecorps/pcsa/get_help_now/ContactOtherStaff.java

```diff
@@ -50,9 +48,9 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
         contactPostStaff.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View v) {
-                //Swapping ContactPostStaff into the fragment container dynamically
-                Fragment contactPostStaff = new ContactPostStaff();
-                MainActivity.swapFragmentIn(getActivity(),contactPostStaff,ContactPostStaff.TAG);
+                //Removing ContactOtherStaff from the fragment container dynamically
+                FragmentManager fragmentManager = getActivity().getSupportFragmentManager();
+                fragmentManager.popBackStack(ContactOtherStaff.TAG,FragmentManager.POP_BACK_STACK_INCLUSIVE);
```

Comment:
This is interesting. 
Currently seeing the onclick listener for a button which takes you to Contact Post Staff from Contact Other Staff. Earlier, I was recreating the fragment and it was unnecessarily adding to the backstack. 
Now, I'm just popping ContactOtherStaff from backstack